build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -12,12 +12,75 @@ import { PackageJson } from "../common/npmPackage";
 import { buildTestCase, TestCaseTypeData } from "../typeValidator/testGeneration";
 import { getFullTypeName, getNodeTypeData, TypeData } from "../typeValidator/typeData";
 import { typeOnly } from "./compatibility";
+import { ExtractorConfig } from "@microsoft/api-extractor";
 
 // Do not check that file exists before opening:
 // Doing so is a time of use vs time of check issue so opening the file could fail anyway.
 // Do not catch error from opening file since the default behavior is fine (exits process with error showing useful message)
 const packageObject: PackageJson = readJsonSync("package.json");
+const previousPackageName = `${packageObject.name}-previous`;
+
+const previousBasePath = `./node_modules/${previousPackageName}`;
```

Comment:
Rather than using hard-coded `/`s, I would recommend using `path.join` for creating these paths. E.g.
```suggestion
const previousBasePath = path.join("node_modules", previousPackageName);
```